tests/e2e/config/jest.setup.js

```diff
@@ -11,18 +11,7 @@ import {
 } from '@wordpress/e2e-test-utils';
 
 // Set the default test timeout.
-let jestTimeoutInMilliSeconds = 60000;
-
-// When running test in the Development mode, the test timeout is increased to 2 minutes which allows for errors to be inspected.
-// Use `await jestPuppeteer.debug()` in test code to pause execution.
-if (
-	process.env.JEST_PUPPETEER_CONFIG ===
-	'tests/e2e/config/jest-puppeteer.dev.config.js'
-) {
-	jestTimeoutInMilliSeconds = 120000;
-}
-
-jest.setTimeout( jestTimeoutInMilliSeconds );
+jest.setTimeout( 60000 );
```

Comment:
I removed some unused config files and these constants. Nothing seems to be using them.